src/engraving/dom/slur.cpp

```diff
@@ -153,11 +153,11 @@ bool SlurSegment::edit(EditData& ed)
         } else {
             if (start && sl->isIncoming()) {
                 sl->undoSetIncoming(false);
-                cr = prevChordRest(e);
+                cr = prevChordRest(e, false, true, true);
```

Comment:
I'm probably being pedantic here, but these kind of calls become quite ugly, because all those false - true - true have zero meaning to the reader unless you open another file and look at the function definition. Ideally, we would create something like a `struct ChordRestNavigateOptions` which contains these options (and give the struct default initializations but _not_ a constructor, that way on the user side the relevant options must be overridden explicitely without using magic true false true things).